eng/targets/Helix.props

```diff
@@ -22,7 +22,7 @@
     <HelixUseArchive>false</HelixUseArchive>
     <LoggingTestingDisableFileLogging Condition="'$(IsHelixJob)' == 'true'">false</LoggingTestingDisableFileLogging>
     <NodeVersion>10.15.3</NodeVersion>
-    <AppRuntimeVersion>5.0.0-ci</AppRuntimeVersion>
+    <AppRuntimeVersion>6.0.0-ci</AppRuntimeVersion>
```

Comment:
Note to self: If we need another update, use `$(AspNetCoreMajorMinorVersion).$(AspNetCorePatchVersion)` here

Reply:
Property removed instead